### PR TITLE
FEA : Code component

### PR DESCRIPTION
**Pull request by marco-souza**
**WARNING**: First approve the PR #67.

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -1 +1,4 @@
-test: "ola maluco doido"
\ No newline at end of file
+test: "ola maluco doido"
```
Comment (brunomacf) [inline config/default.yml]: I guess this is not necessary anymore :)

File: src/code/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import classNames from "classnames";
+import styles from "./styles";
+import Prism from "prismjs";
+import "prism-themes/themes/prism-atom-dark.css";
+
+type Langs = "text" | "javascript"
+type Props = {
```
Comment (brunomacf) [inline src/code/index.js]: Let's provide some docs here.... do you know how to document type in jsdocs world (like struct types)?

File: src/text/index.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import React from "react";
-import {LoggerFactory} from "darch/src/utils";
+import {LoggerFactory} from "utils";
 
-const Logger = new LoggerFactory("text", {level:"error"});
+const Logger = new LoggerFactory("text");
 type Props = {
```
Comment (brunomacf) [inline src/text/index.js]: Let's provide some docs here either :)

File: config/default.yml (hunk 1 above)
Comment (marco-souza) [reply config/default.yml]: It's already removed in another branch.

**Pull request merged**